flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```

Comment:
why no support for aarch64?

Reply:
You could build from source instead.

Reply:
Is this a blocking issue or can the manifest be accepted as-is and changed to building from sources later? I am asking because setting up build from sources is probably not trivial.